mps_youtube/players/vlc.py

```diff
@@ -31,3 +36,13 @@ def launch_player(self, cmd):
 
     def _help(self, short=True):
         pass
+
+    def _kill_instance(self):
```

Comment:
For a process spawned from Python, one may want to use `Popen.terminate` rather than trying to use killall. That should be portable and not kill another vlc instance if it happened to be running.